modules/multi_step/variables.tf

```diff
@@ -51,7 +51,7 @@ variable "alert_tags_separator" {
 }
 
 variable "options_list" {
-  type = map(any)
+  type = any
```

Comment:
This is the breaking change. Why? TF is expecting all `var.options_list` attributes to be the same type:

![image](https://github.com/ohpensource/terraform-datadog-ohp-synthetics/assets/97619732/e54d47f9-4fc8-4930-81f0-391e1679b80c)

The property `options_list.scheduling`  is a map with different attributes as: 
* `var.options_list.scheduling.timezone`
* `var.options_list.scheduling.timeframes`

This force us to change `var.options_list` to `type = any`